validator.js

```diff
@@ -40,191 +40,6 @@ function _typeof(obj) {
   return _typeof(obj);
 }
 
-function _toArray(arr) {
```

Comment:
Remove these changes.

Reply:
See 330ad51. Not sure why they're on master.